### ISSUE TITLE
plio cannot write out 'GoodnessOfFit' value

### ISSUE THREAD

**Issue by ladoramkershner** [USGS-Astrogeology/plio#144]
After subpixel registration using pointreg, a purely ISIS control network will contain the following keywords:
```
MinimumPixelZScore = -5.5790822905078
MaximumPixelZScore = 7.5094157155466
GoodnessOfFit      = 0.9185360514395
```

However, none of these keywords appear in ControlPointFileEntryV0005_pb2.py. So you cannot use plio to write out a subpixel registered network that contains your matching criteria.

**Comment by jessemapel** [comment]
These values are stored in the ControlMeasureLogData which is a rather odd part of ISIS ControlMeasures and is not handled well in PLIO right now:

https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/control/objs/ControlMeasureLogData/ControlMeasureLogData.h#L63

**Comment by jessemapel** [comment]
In the protobuf, the ControlMeasureLogData is stored as an array of messages in each ControlMeasures message (the same way that ControlMeasures are stored as an array of messages in each ControlPoint message).

https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/control/objs/ControlNetVersioner/ControlPointFileEntryV0005.proto#L84

**Comment by jlaura** [comment]
@jessemapel Does ISIS not define a ControlPointLogData object? I see PointLogData defined in the protobuf spec, but I do not see anywhere that ISIS is making use of the field. I'm looking for the enum of the data type to field like you linked above for the measures.

**Comment by jessemapel** [comment]
Looking through the code base, I can't see anything that actually uses it. I believe it's an old carry-over from before control measures had log data.

In short, it can be ignored.

**Comment by jlaura** [comment]
Fixed in #146.
